### ISSUE TITLE
Investigate role on Status tags

### ISSUE THREAD

**Issue by ASM995** [mi6/ic-ui-kit#20]
## Summary of the bug
Discovered during accessibility testing on 06/12/2021:
The status tag component has a role property but this does not appear to be read out by assistive technology. Instead, only the text passed in the 'label' property is read out. 
The component requires either an aria-label or equivalent to make sure that the context, i.e. 'Status/Status tag' or similar, is read out by assistive technology.

## 🪜 How to reproduce
Use a screen reader and navigate to a status tag. 

## 🧐 Expected behaviour

*Given* I am using assistive technology (i.e. a screen reader)
*When* I navigate to a status tag
*Then* I expect to be told I am on a status tag in addition to being given the label text. Screen reader: "Status. [visible label]."

We can use the accessible label to append "Status. "+[visible label]. Take care to ensure the WCAG label in name criteria is met here. 

Additionally, we should provide functionality to allow status tags to act as a live region, or not. By default, status tags seem to have a role of status, which creates a live region for the status tag. However, status tags are not always used as live regions; they may be static on a page. In this case, the status tag would not need to have role=Status as it is not changing on the page. Only status's that can change dynamically whilst a user in on the page should have a role=Status and therefore a live region. 

Suggestion is to provide a prop that allows a user to specify whether or not they want the status tag to act as a 'status' and therefore be a live region. 

**Given** that a developer is implementing a status tag that is static on a page, 
**When** setting its props, 
**Then** they can chose to turn off the status role, thereby removing the live region. 

**Given** that a developer is implementing a status tag that will update on a page, 
**When** setting its props, 
**Then** they can chose to turn on the status role, thereby creating a live region. 

## 📸 Screenshots or code
If you can, add screenshots to show the problem. If you need to show a code example, you can include a snippet or link to a repository with an example.

## 🖥 Desktop
 - OS: [e.g. macOS]
 - Browser: [e.g. chrome, safari]
 - Version: [e.g. 22]

## 📱 Mobile
 - Device: [e.g. iPhone13]
 - Browser: [e.g. chrome, safari]
 - Version: [e.g. 22]

## Additional info
Tell us anything else useful to help us fix or understand the problem.

**Comment by ASM995** [comment]
More investigation required to determine exactly what we want to do @DBD324
